### DISCUSSION TITLE
gluetun and qbitorrent suddenly not working - Newbie sorry

### DISCUSSION THREAD

**Discussion by Belgregor71** [qdm12/gluetun#1425] [Q&A] [answered]
Hi,

I've had gluetun and qbitorrent working great  for me for the last couple of months on my synology NAS.   

Had a powerfail last night and gluetun keeps stopping.  Sorry if it's something obvious.
[gluetun.csv](https://github.com/qdm12/gluetun/files/10837386/gluetun.csv)

**Comment by qdm12** [comment, marked as answer]
Just repull the latest image `docker pull qmcgaw/gluetun`. I accidentally broke it yesterday, see #1421
